applications/GeoMechanicsApplication/tests/cpp_tests/test_damping_matrix.cpp

```diff
@@ -0,0 +1,58 @@
+// KRATOS___
+//     //   ) )
+//    //         ___      ___
+//   //  ____  //___) ) //   ) )
+//  //    / / //       //   / /
+// ((____/ / ((____   ((___/ /  MECHANICS
+//
+//  License:         geo_mechanics_application/license.txt
+//
+//  Main authors:    Gennady Markelov
+//
+
+#include "custom_utilities/equation_of_motion_utilities.h"
+#include "includes/checks.h"
+#include "testing/testing.h"
+#include <boost/numeric/ublas/assignment.hpp>
```

Comment:
I don't see assignment being used, could we remove this include?

Reply:
Thank you. It is removed.